Header header
uint8 channel
---
uint8 error # 0 bei ok, != 0 sonst
